R/mobr.R

```diff
@@ -109,7 +109,7 @@ plot.mobr = function(mobr, group = NULL, par_args=NULL,
       ylim = range(lapply(mobr$discrete, function(x)
                           lapply(x[ , -(1:2)], function(y)
                                  as.numeric(as.character(y)))))
-    for (i in 3:1){
```

Comment:
If I remember correctly this change was based on Jon's comment that empirical ecologists tend to think in the order of accumulation - sample - individual.